AndroidProject/app/src/main/java/com/example/androidproject/RecyclerAdapter.java

```diff
@@ -0,0 +1,47 @@
+package com.example.androidproject;
+
+import android.graphics.drawable.Drawable;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+
+import androidx.recyclerview.widget.RecyclerView;
+
+public class RecyclerAdapter extends RecyclerView.Adapter<RecyclerAdapter.ImageViewHolder> {
+
+    private Drawable[] drawables;
+
+    RecyclerAdapter(Drawable[] drawables) {
+        this.drawables = drawables;
+    }
+
+    static class ImageViewHolder extends RecyclerView.ViewHolder {
+
+        ImageView photo;
+
+        ImageViewHolder(View itemView) {
+            super(itemView);
+            photo = itemView.findViewById(R.id.photo);
+        }
+    }
+
+    @Override
+    public ImageViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.album_layout, parent, false);
+        ImageViewHolder imageViewHolder = new ImageViewHolder(view);
+
+        return imageViewHolder;
```

Comment:
Can be inline `return new ImageViewHolder(view);`

Reply:
done